src/spline2/MultiBsplineStd.hpp

```diff
@@ -35,7 +35,7 @@ namespace qmcplusplus
   template<typename T>
     inline void 
     MultiBspline<T>::evaluate_vgl_impl(T x, T y, T z, 
-        T* restrict vals, T* restrict grads, T* restrict lapl, int first, int last, size_t out_offset) const
+        T* restrict vals, T* restrict grads, T* restrict lapl, size_t out_offset) const
```

Comment:
out_offset was changed to out_stride elsewhere

Reply:
Nevermind.  I see it really is an offset here.

Reply:
Let me check. I probably changed one place but forgot another. What is a better out_offset or out_stride?

Reply:
Changed back to out_offset.

Reply:
out_stride implies the fastest dimension, at least to me.  out_offset seems better, as it is the distance between the SoA elements.